.github/workflows/compare-helm-with-jsonnet.yml

```diff
@@ -10,13 +10,40 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - name: Get build image from Makefile
+        id: build_image_step
+        run: echo "build_image=$(make print-build-image)" >> "$GITHUB_OUTPUT"    
+    outputs:
+      build_image: ${{ steps.build_image_step.outputs.build_image }}
+
+  goversion:
+    runs-on: ubuntu-latest
+    needs: prepare
+    container: 
+      image: ${{ needs.prepare.outputs.build_image }}
+    steps:
+      - name: Get Go Version
+        id: go-version
+        run: |
+          version=$(go version | awk '{print $3}' | sed 's/go//')
```

Comment:
here we can't really run make print-go-version, knowing in github workflow, spin up a docker image inside is not really supported, we need to run mimir-build-image to get the go version. and it is what is doing here. but there is no make target in mimir-build-image itself.